app/models/comment.rb

```diff
@@ -1,4 +1,4 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
   belongs_to :post
 end
```

Comment:
- [ ] nice work. please kindly ensure the Comment model has a method that updates the comments counter for a post. you can use the `after save` to call the method as explained above
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```